### ISSUE TITLE
hasMedia always is false! 2.2104.9

### ISSUE THREAD

**Issue by rasoolhp** [pedroslopez/whatsapp-web.js#541]
### Bug description
today I get hasMedia:false for all messages!!!
and downloadMedia always returns undefined
my app was ok for several months! but today... no :(

### Reproduction steps
1. get chat
2. get messages
3. download a message with photo
...

### Expected behavior 
```
Error: Evaluation failed: Object
    at ExecutionContext._evaluateInternal (/Users/rasoolhp/Desktop/watbotwo/node-js-getting-started/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:217:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async ExecutionContext.evaluate (/Users/rasoolhp/Desktop/watbotwo/node-js-getting-started/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:106:16)
    at async Message.downloadMedia (/Users/rasoolhp/Desktop/watbotwo/node-js-getting-started/src/structures/Message.js:231:24)
```

### Relevant code
#465
#298

### Environment (please complete the following information):
**WhatsApp**
 - Account type:Standard
 - Device OS:Android
 - WhatsApp Web version: 2.2104.9

**Library**
 - Browser Chromium:
 - NodeJS version: v12.18.2
 - npm or yarn version:6.14.8
 - whatsapp-web.js version: test on 1.11.1 and 1.12.4

**Comment by rmazzine** [comment]
Got the same problem. The object to receive media has changed, therefore the following line was added to circumvent that 

https://github.com/pedroslopez/whatsapp-web.js/blob/204a1c79af306218aef2417dd4b9ea07a3b7ee71/src/structures/Message.js#L256

And this one was changed:

https://github.com/pedroslopez/whatsapp-web.js/blob/204a1c79af306218aef2417dd4b9ea07a3b7ee71/src/structures/Message.js#L258

So, you can just make those changes or update the package to the most recent version and I think it will work!

**Comment by pedroslopez** [comment]
As mentioned, this was fixed on 1.12.3 and 1.12.4, so please make sure you've updated properly and are running the latest version.
You can run this snippet to verify the versions:

```js
client.on('ready', async () =>{
    console.log("WhatsApp Web v", await client.getWWebVersion());
    console.log("WWebJS v", require("whatsapp-web.js").version);
});
```

**Comment by rasoolhp** [comment]
WhatsApp Web v 2.2104.9
WWebJS v 1.12.4

I still have the problem

**Comment by rasoolhp** [comment]
I think my update process is ok.
<img width="1680" alt="Screen Shot 1399-11-26 at 03 33 06" src="https://user-images.githubusercontent.com/17994452/107864864-f4124880-6e75-11eb-9a60-fc13825cdd2c.png">

@pedroslopez e2acd03efcd938721dc57d4de082bb2378e9c19c

in his commit, it seems debugged.

but it's for v2.2104.6 and not 2.2104.9

can you check it with v2.2104.9?

or still, my update process is wrong, tell me.

(even I delete the lock file and node module folder and reinstall them again)

**Comment by rasoolhp** [comment]
its doc but it have hasMedia: false!!!:
<img width="631" alt="image" src="https://user-images.githubusercontent.com/17994452/107874004-d2dc4700-6ecb-11eb-85b0-7bfbe324b3d3.png">


its img but it have hasMedia: false!!!:
<img width="631" alt="image" src="https://user-images.githubusercontent.com/17994452/107874059-1d5dc380-6ecc-11eb-95ee-b08d5bdec39c.png">

for all messages! hasMedia is false :(

really why?

**Comment by rasoolhp** [comment]
and when I set it true force
 I get this error:
```js
Error: Evaluation failed: Object
    at ExecutionContext._evaluateInternal (/Users/rasoolhp/Desktop/watbotwo/node-js-getting-started/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:217:19)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async ExecutionContext.evaluate (/Users/rasoolhp/Desktop/watbotwo/node-js-getting-started/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:106:16)
    at async Message.downloadMedia (/Users/rasoolhp/Desktop/watbotwo/node-js-getting-started/src/structures/Message.js:231:24)
```

**Comment by rmazzine** [comment]
Strange.

`hasMedia` is defined in this line: 
https://github.com/pedroslopez/whatsapp-web.js/blob/204a1c79af306218aef2417dd4b9ea07a3b7ee71/src/structures/Message.js#L43

Try to insert bellow (on line 44) that `console.log(JSON.stringify(data))`

And check if there's a key for `deprecatedMms3Url` or `clientUrl` with a URL. If you post this object here, be careful with sensitive data.

**Comment by rasoolhp** [comment]
I inserted that. but not worked.
I added a simple log on several files of node modules...
but they didn't work.
how can I do it? what is my wrong?

**Comment by rasoolhp** [comment]
it has deprecatedMms3Url

**Comment by rasoolhp** [comment]
but still, hadMedia is false!!! omg!!!
i will try to install node package directly from Github!

**Comment by pedroslopez** [comment]
@rasoolhp Check your discord DMs

**Comment by rasoolhp** [comment]
ok, guys.
my problem solved 5 days ago.
I don't know how!!!
but now I have an "src" folder on my project root! and it has the codes of this repo.

it's bad and dirty but it works these days...
for the next update, I will find a better way.

**Comment by May-Aronov** [comment]
> As mentioned, this was fixed on 1.12.2 and 1.12.4, so please make sure you've updated properly and are running the latest version.
> You can run this snippet to verify the versions:
> 
> ```js
> client.on('ready', async () =>{
>     console.log("WhatsApp Web v", await client.getWWebVersion());
>     console.log("WWebJS v", require("whatsapp-web.js").version);
> });
> ```

Is this fixed also for v 1.12.5?

I recently check it with the latest version of WWebJS and Whatsapp web version 2.2106.5,
it's still happening just when the client tries uploading media that got pictured inside the chat by the "image icon".

**hasMedia false is received on media_uploaded event.
